_data/projects/eBooker.yml

```diff
@@ -1,11 +1,13 @@
 name: eBooker
-desc: eBooker is a command-line application written in Python. With it, you don't have to learn programming or manage massive user interfaces to make great eBooks. We provide simple download scripts for all major operating systems, so just download, install, and enjoy.
+desc: eBooker is a command-line tool which lets you create ebooks and other text files from command line with ease. You don't have to be a programming expert or a nerd to use this. Anyone with a basic knowledge in computers can use this tool very easily. We provide just one script to run, so run it! So far, it can execute simple commands and also create and edit a file.
```

Comment:
This is still going into technical details. Can we drop this bit and focus on more on the "elevator pitch"?

> We provide just one script to run, so run it! So far, it can execute simple commands and also create and edit a file.